treeherder/webapp/api/performance_data.py

```diff
@@ -819,6 +820,12 @@ def list(self, request):
                 base_stddev_pct = self._get_stddev_pct(base_avg_value, base_stddev)
                 new_stddev_pct = self._get_stddev_pct(new_avg_value, new_stddev)
 
+                sig_hash = (
+                    base_sig.get('signature_hash', '')
```

Comment:
Here you don't have access to the `signature_hash` because you haven't added it in values returned by the `_get_signatures_values` method.